### ISSUE TITLE
it’s encoding with non English characters

### ISSUE THREAD

**Issue by chenpaner** [SpectralVectors/Keymapper#1]
Traceback (most recent call last):
  File "C:\Users\CP\AppData\Roaming\Blender Foundation\Blender\Big addons\addons\Keymapper-main\Properties.py", line 16, in search
    if keymap_item.name == self.keymap_item:
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 1-2: invalid continuation byte
![Snipaste_2023-08-29_15-12-57](https://github.com/SpectralVectors/Keymapper/assets/107256886/63d21ab4-44cb-4d77-a3d3-e3d2fdeed826)

Here's a post where I found similar issues that might help you fix this bug!
https://devtalk.blender.org/t/enumproperty-and-string-encoding/7835

**Comment by SpectralVectors** [comment]
Confirming the bug - seems to affect all non-English languages

**Comment by SpectralVectors** [comment]
Hi chepaner, I've fixed the issue.

For the curious:
I'm using two lists inside EnumProperties: keymaps and keymap_items
Though they both work, the references to the data they contain can get lost, resulting in the above issue.
To fix it I registered two empty lists as Scene properties and copied the keymaps and keymap_items lists to them before returning the lists to their EnumProperties.
It's a workaround for a known bug, but it took me a few tries to figure out exactly what to do.
The relevant changes are found in the __init__ and Properties files.

Thanks for bringing the issue to my attention, and for supplying a link to help troubleshoot it!
